src/Dictionaries/MySQLDictionarySource.cpp

```diff
@@ -119,67 +119,43 @@ std::string MySQLDictionarySource::getUpdateFieldAndDate()
     }
 }
 
-BlockInputStreamPtr MySQLDictionarySource::retriedCreateMySqlBIStream(const std::string & data_fetch_query_str, const size_t max_tries)
+BlockInputStreamPtr MySQLDictionarySource::loadBase(const String & query)
```

Comment:
The name is not clear

Reply:
I made its name the same as it was made for XDBCDictionarySource, and then in PostgreSQL in another PR. So here it should better also be the same.